packages/bits/src/lib/wizard-v2/wizard-horizontal/wizard-horizontal.component.ts

```diff
@@ -13,7 +13,7 @@ import { WizardDirective } from "../wizard.directive";
     host: {
         "class": "nui-wizard-horizontal-layout",
         "[class.nui-wizard-label-position-end]": "labelPosition == 'end'",
-        "[class.nui-wizard-label-position-bottom]": "labelPosition == 'bottom'",
+        "[class.nui-wizard-label-position-top]": "labelPosition == 'top'",
```

Comment:
For BEM, these should really be: `nui-wizard--step-label-position-end` and `nui-wizard--step-label-position-top`.

But, I don't understand why we're applying a class to the entire wizard host just for the step labels.